go/vt/vttablet/tabletmanager/rpc_actions.go

```diff
@@ -33,6 +33,16 @@ import (
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
+// DBAction is used to tell ChangeTabletType whether to call SetReadOnly on change to
```

Comment:
I debated not exporting this enum. However, since it is a function argument to `ChangeTabletType` I felt that it must be exported.